server/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -72,27 +80,34 @@
 public class MTree implements Serializable {
 
   private static final long serialVersionUID = -4200394435237291964L;
+
   private MNode root;
+  private int snapshotLineNumber;
```

Comment:
Actually, it means the line number of mlog file that is serialized in snapshot. And the current name may be confusing because it's referred to the line number in snapshot log file. So you'd better change rename this variable as `serializedLineNumber`.

Reply:
snapshotedLineNumber is ok...

Reply:
Both are good for me. @samperson1997 think mine is better. :D

Reply:
I'll just take `snapshotedLineNumber` : )

Reply:
@qiaojialin the past participle of _snapshot_ is _snapshot_ ... When I changed it to _snapshoted_ or _snapshotted_, the IDE will just give a spelling error hint:
![image](https://user-images.githubusercontent.com/19167280/84997891-b1cb2700-b181-11ea-9153-263e396f44e4.png)

Reply:
snapshot is ok

Reply:
add some comment

Reply:
Added